TeachingRecordSystem/tests/TeachingRecordSystem.TestCommon/TestData.CreatePerson.cs

```diff
@@ -211,6 +235,12 @@ public CreatePersonBuilder WithQts(DateOnly? qtsDate = null)
             return this;
         }
 
+        public CreatePersonBuilder WithQtlsDate(DateOnly? qtlsDate)
+        {
+            _qtlsDate = qtlsDate;
```

Comment:
This should have a check that value hasn't already been assigned and is different, like `WithEmail`, `WithMobileNumber` etc